From b10c083e9b9de46b54873780f73dce57fa1b6d4f Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Tue, 5 Sep 2017 14:53:25 +0200
Subject: [PATCH] units: order container and console getty units after
 getty-pre.target

(cherry picked from commit 45e27532971ac84e835a2879df510a581f933fcd)

Related: #1173080
---
 units/console-getty.service.m4.in    | 2 +-
 units/container-getty@.service.m4.in | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/units/console-getty.service.m4.in b/units/console-getty.service.m4.in
index 413d94094..61ecf8951 100644
--- a/units/console-getty.service.m4.in
+++ b/units/console-getty.service.m4.in
@@ -11,7 +11,7 @@ Documentation=man:agetty(8)
 After=systemd-user-sessions.service plymouth-quit-wait.service
 ConditionPathExists=/dev/console
 m4_ifdef(`HAVE_SYSV_COMPAT',
-After=rc-local.service
+After=rc-local.service getty-pre.target
 )m4_dnl
 Before=getty.target
 
diff --git a/units/container-getty@.service.m4.in b/units/container-getty@.service.m4.in
index e126f3a48..4395ef5ce 100644
--- a/units/container-getty@.service.m4.in
+++ b/units/container-getty@.service.m4.in
@@ -10,7 +10,7 @@ Description=Container Getty on /dev/pts/%I
 Documentation=man:agetty(8) man:machinectl(1)
 After=systemd-user-sessions.service plymouth-quit-wait.service
 m4_ifdef(`HAVE_SYSV_COMPAT',
-After=rc-local.service
+After=rc-local.service getty-pre.target
 )m4_dnl
 Before=getty.target
 IgnoreOnIsolate=yes
